eng/Directory.Build.Common.targets

```diff
@@ -198,6 +199,12 @@
     <Compile Include="$(AzureCoreSharedSources)AzureResourceProviderNamespaceAttribute.cs" LinkBase="Shared" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and '$(IsTestProject)' != 'true'">
```

Comment:
@AlexanderSher can you please move this into individual project files after the PR goes in.

Reply:
Ok